### ISSUE TITLE
colIQRs reports error when x is an one-column matrix

### ISSUE THREAD

**Issue by everdark** [HenrikBengtsson/matrixStats#36]
Hi, 
I've found that `colIQRs` behaves differently from `colMedians` when its matrix argument has only one column. Specifically, `colMedians` can deal with one-column matrix as expected, while `colIQRs` reports error as this:
```
Error in Q[, 2L, drop = TRUE] : incorrect number of dimensions
```

A simple script to test about this issue:
```
mm <- matrix(rnorm(20), 10, 2)
identical(colMedians(mm), apply(mm, 2, median)) # TRUE
identical(colIQRs(mm), apply(mm, 2, IQR)) # TRUE

mm <- matrix(rnorm(10), 10, 1)
identical(colMedians(mm), median(mm)) # TRUE
colIQRs(mm) # error
```

I am using 0.14.0. 
```
> sessionInfo()
R version 3.2.0 (2015-04-16)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.3 (Yosemite)
```

**Comment by HenrikBengtsson** [comment]
Thank you for the report.  This bug was actually fixed yesterday (commit db093ae by @Dongcan-Jiang):

o BUG FIX: rowQuantiles() and rowIQRs() did not work for single-row matrices.  Analogously for the corresponding column functions.

Until we push out next version of matrixStats to CRAN, you can install the developers version (0.14.0-9000) of this package by:
```r
source("http://callr.org/install#HenrikBengtsson/matrixStats@develop")
```
(depending on your setup, there might be quite a few extra package installed in order to be able to install from GitHub like this).  Let me know if this doesn't work.

**Comment by everdark** [comment]
It works! Thanks for the update.
